### DISCUSSION TITLE
Amazon Warehousing and Distribution API

### DISCUSSION THREAD

**Discussion by NaufragarMeDolce** [abuzuhri/Amazon-SP-API-CSharp#761] [Q&A]
I am a seller and code my own API because I have that skill form a previous life.  I am in need of Amazon Warehousing and Distribution API functionality but I don't see it here in any of the code, issues or discussions.  Did I miss something?  Are there plans for this in the SDK? I am actively sending AWD shipments but they are currently going into an inventory-control black hole since there are no API hooks.

**Comment by clementcampagna** [comment]
Hi,
You should take a look at this if you haven't done so already: https://developer-docs.amazon.com/sp-api/lang-en_EN/docs/awd-2024-05-09-model

**Reply by NaufragarMeDolce** [reply]
Hi, 

Yes, I saw that.  I'm not good with REST though else I wouldn't need the SDK.  Is there a time frame?

**Reply by NaufragarMeDolce** [reply]
So, no news on this?  The tools Amazon provides for tacking AWD are woeful.
